angular-ngrx-scss/src/app/user/services/user.service.ts

```diff
@@ -19,153 +15,109 @@ import { environment } from 'src/environments/environment';
 export class UserService {
   constructor(private http: HttpClient) {}
 
-  getAuthenticatedUserInfo(): Observable<UserState> {
+  /**
+   * Gets details for the authenticated user
+   * Important to note that this is specifically for the currently authenticated user
+   * @returns the full GH response with the user's account info
+   */
+  getAuthenticatedUserInfo(): Observable<UserApiResponse> {
     const url = `${environment.githubUrl}/user`;
 
-    return this.http.get<UserApiResponse>(url).pipe(
-      map((data) => ({
-        avatar: data.avatar_url,
-        bio: data.bio,
-        blog: data.blog,
-        company: data.company,
-        email: data.email,
-        followers: data.followers,
-        following: data.following,
-        location: data.location,
-        name: data.name,
-        twitter_username: data.twitter_username,
-        username: data.login,
-        type: data.type,
-      })),
-    );
+    return this.http.get<UserApiResponse>(url, {
+      headers: {
+        Accept: 'application/vnd.github.v3+json',
+      },
+    });
```

Comment:
same as in other service. Can we move this to an interceptor?

Reply:
Acknowledged in the other comment - created a ticket to do this in a separate PR.